src/yaml.jl

```diff
@@ -1,26 +1,26 @@
 """
-    dftramo_run(filename::AbstractString, software::AbstractString="vasp")
+    dftramo_run(filename::AbstractString)
 
 Automatically runs DFTraMO from a configuration yaml file.
 """
-function dftramo_run(filename::AbstractString, software::AbstractString="vasp")
-    (runs, checkpoint, auto_psphere, dftinfo, emin, emax) = read_run_yaml(filename, software)
-    occ_states = get_occupied_states(dftinfo.wave, emin, emax)
-    super = Supercell(dftinfo.xtal, ORB_DICT)
+function dftramo_run(filename::AbstractString)
+    ramoinput = read_yaml(filename)
+    occ_states = get_occupied_states(Electrum.PlanewaveWavefunction(ramoinput), ramoinput.emin, ramoinput.emax)
+    super = Supercell(Electrum.supercell(ramoinput), ORB_DICT)
     S = make_overlap_mat(occ_states)
     H = generate_H(super, DFTRAMO_EHT_PARAMS)
     
-    if !isnothing(checkpoint)
-        (psi_previous, num_electrons_left, num_raMO) = import_checkpoint(checkpoint)
+    if !isnothing(ramoinput.checkpoint) && length(ramoinput.checkpoint) > 0
+        (psi_previous, num_electrons_left, num_raMO) = import_checkpoint(ramoinput.checkpoint)
     else
-        num_electrons_left = sum([get(E_DICT, n.atom.name, 0) for n in dftinfo.xtal.atoms])
+        num_electrons_left = sum([get(E_DICT, n.atom.name, 0) for n in Electrum.PeriodicAtomList(Electrum.Crystal(ramoinput))])
```

Comment:
There is a constructor for a `PeriodicAtomList` that takes a `raMOInput`, so you can simplify this line to 
```julia
num_electrons_left = sum([get(E_DICT, n.atom.name, 0) for n in PeriodicAtomList(ramoinput)])
```

Reply:
Resolved in [001b98c](https://github.com/xamberl/DFT-raMO/pull/39/commits/001b98c698962603e5cf15614e573730df3ca5ba)